Project-Descriptions-and-Plans/CVE20/CVE20-PA-2024.md

```diff
@@ -0,0 +1,311 @@
+﻿This update to both the 2022 Project Launch (PL) and 2023 Plan Approved (PA) Documentation for the CVE20 project is being proposed for TWG approval by OpenHW Staff. It updates both PL and PA fields. The reasons for the update are that fundamental things have changed with the project.
+- The project team has changed. Verification work is now being driven directly by OpenHW Group staff rather than OpenHW Members, but Members are still encouraged to get involved.
+- The scope of the E20 core project is slightly different. The originally included Core Complex is now deferred to a later version
+- Only a single configuration of the core (RV32IMC) is now included in the verification plan, and other configurations are deferred. 
+- The verification approach now uses Spike as a reference model, and the creation of that model is part of this project.  
+
+## Summary of project
+  The CV32E20 proposed project develops a TRL5, area-efficient 2-stage microcontroller core based on Ibex 
+as part of the CORE-V family of cores
+
+
+  The 2021 scope of the project included of detailing which of the Ibex parameters are being removed, or verified, or neither verified or removed (left for future work). This has been largely completed as of 2024
+
+In 2021, completing design enhancements and integration of the interfaces into the core complex was planned. In 2024, most of the core complex aspects have been removed or simplified.
+
+Completing verification, including creation of a Spike reference model, and documentation remains a key aspect of the project. 
+
+## PL field: Components of the Project
+### Component 1a "RTL design of the core". Note, this list constitutes the PA feature specification. 
+- RTL design of the Core starting from Ibex with the following features:
+ - I: Base Integer Instruction Set, with 32, 32-bit general-purpose registers
+ - M: Standard Extension for Integer Multiplication and Division
+ - C: Standard Extension for Compressed 16-bit Instructions
+ - Machine Mode 
+ - Cleaning Ibex RTL of unused parameters 
+ - Exposing privilege pins (privilege CSR bits are exposed as bus attribute signals)
+ - Modification to make OBIv1.2-compliant bus interfaces.
+ - Adding and extending the rvfi interface
+ - CV32E40P-like Sleep unit
+ 
+- No longer included in scope as of 2024:
```

Comment:
This list repeats much of what "Removed from project scope as of 2024:" under the "Component 1b" heading below.

Reply:
i found 1 duplication and removed it